board/fancom/hmi/MAINTAINERS

```diff
@@ -0,0 +1,6 @@
+RPI BOARD
```

Comment:
Copy paste? we're the maintaners right?

Reply:
Yes this was from copy/paste. I disregarded it but will fix this.